internal/query/range_experiment.go

```diff
@@ -0,0 +1,55 @@
+//go:build go1.22 && goexperiment.rangefunc
+
+package query
+
+import (
+	"context"
+	"io"
+	"iter"
+
+	"github.com/ydb-platform/ydb-go-sdk/v3/internal/xerrors"
+	"github.com/ydb-platform/ydb-go-sdk/v3/query"
+)
+
+type (
+	resultRange struct {
```

Comment:
The struct same as for file without experimental. What if remove conditions for range.go and implement functions only within experimental func

Reply:
replaced with static funcs